### PR TITLE
Add unit tests for bls utils

### PR DESCRIPTION
**Pull request by rlan35**

### CODE REVIEW

File: internal/utils/utils_test.go [hunk 1]
```diff
@@ -48,6 +48,14 @@ func TestGenKey(t *testing.T) {
 	GenKey("3.3.3.3", "3456")
 }
 
+// Test for GenKeyBLS
+func TestGenKeyBLS(t *testing.T) {
+	priKey, pubKey := GenKeyBLS("3.3.3.3", "3456")
+	if priKey == nil || pubKey == nil {
```
Comment (LeoHChen) [inline internal/utils/utils_test.go]: can you find a way to compare the key generated is correct or not?

File: crypto/bls/bls_test.go [hunk 2]
```diff
@@ -0,0 +1,37 @@
+package bls
+
+import (
+	"testing"
+
+	"github.com/harmony-one/bls/ffi/go/bls"
+	"github.com/harmony-one/harmony/internal/utils"
+)
+
+func TestNewMask(test *testing.T) {
```
Comment (LeoHChen) [inline crypto/bls/bls_test.go]: please add a comment for this function.

File: internal/utils/utils_test.go (hunk 1 above)
Comment (rlan35) [reply internal/utils/utils_test.go]: There is not much special logic in the function. It's basically putting what is generated from GetUniqueIDFromIPPort as bytes into private key and call the GetPublicKey util of BLS. I just add result checking for it based on the output. But we are trusting the BLS package is working fine for now.

**Comment by rlan35** [comment]
> what's the test coverage for bls package?

ok  	github.com/harmony-one/harmony/crypto/bls	0.038s	coverage: 43.5% of statements

**Pull request merged**